djs-bot/commands/music/autoleave.js

```diff
@@ -56,7 +56,9 @@ const command = new SlashCommand()
       }`
     );
 
-    return interaction.reply({ embeds: [autoLeaveEmbed] });
+    const ret = await interaction.reply({ embeds: [autoLeaveEmbed], fetchReply: true });
+    if (ret) setTimeout(() => ret.delete().catch(client.warn), 20000);
```

Comment:
I see many repetition of this, can be made into utils

Reply:
Agree, this should be done in the future